Segment_Delaunay_graph_2/include/CGAL/Segment_Delaunay_graph_2/in_place_edge_list.h

```diff
@@ -243,7 +243,7 @@ class In_place_edge_list_for_sdg {
 
   void remove(const Edge& e) {
     CGAL_precondition( is_in_list(e) );
-    static Edge SENTINEL_QUEUE_EDGE = Edge(Face_handle(), -1);
+    static const Edge SENTINEL_QUEUE_EDGE = Edge(Face_handle(), -1);
```

Comment:
I do not think that this initialization of a `static const` variable in a scope is guaranteed thread-safe, before C++11.